### PR TITLE
feat(confirmsendmodal): validate ln-amount is smaller than balance #64

### PR DESCRIPTION
**Pull request by escapedcat**
- [x] expiry date of the invoice has passed
- [x] amount of the invoice is higher than the available balance

### CODE REVIEW

File: src/components/Shared/SendModal/ConfirmSendModal/ConfirmSendModal.tsx [hunk 1]
```diff
@@ -9,17 +9,31 @@ import { ReactComponent as ChevronLeft } from "../../../../assets/chevron-left.s
...
+  const { address, back, balance, close, comment, fee, invoiceAmount, ln } =
+    props;
 
   const [amount, setAmount] = useState(0);
 
   const amountChangeHandler = (event: ChangeEvent<HTMLInputElement>) => {
     setAmount(+event.target.value);
   };
 
+  const isInvoiceAmountBiggerThanBalance = invoiceAmount > balance;
```
Comment (escapedcat) [inline src/components/Shared/SendModal/ConfirmSendModal/ConfirmSendModal.tsx:35]: Should this be an effect? Not too familiar with react in this case

Comment (cstenglein) [reply src/components/Shared/SendModal/ConfirmSendModal/ConfirmSendModal.tsx:35]: Shouldn't be an issue; ConfirmSend won't re-render that much IMO.

File: backend-mock/index.js [hunk 2]
```diff
@@ -231,7 +231,7 @@ app.get("/api/v1/lightning/decode-pay-req", (req, res) => {
       payment_hash:
         "dc171b0d9a6c33d40ba2d9ed95819b29af40d83132b15072ab4e8b60feb08b90",
       num_satoshis: 20,
-      timestamp: 1633322501,
+      timestamp: 1893456000000,
```
Comment (escapedcat) [inline backend-mock/index.js:234]: Mock invoice expires in 2030

**Comment by cstenglein** [comment]
Due to my change, now it has a merge conflict, sorry @escapedcat  :(

File: src/components/Shared/SendModal/ConfirmSendModal/ConfirmSendModal.test.tsx [hunk 3]
```diff
@@ -8,26 +8,29 @@ import ConfirmSendModal from "./ConfirmSendModal";
...
-  ln: true,
+  invoiceAmount: 0,
+  isLnTx: true,
+  timestamp: 1893456000000, // 01 Jan 2030 00:00:00 GMT
 };
 
 describe("ConfirmSendModal", () => {
   describe("ln-invoice with zero amount", () => {
-    beforeEach(() => {
+    const setup = () =>
```
Comment (escapedcat) [inline src/components/Shared/SendModal/ConfirmSendModal/ConfirmSendModal.test.tsx:24]: eslint complains: https://github.com/testing-library/eslint-plugin-testing-library/blob/main/docs/rules/no-render-in-setup.md

**Review by cstenglein — APPROVED** [review]
Thank you! :)

**Pull request merged**